driver/XDMA/linux-kernel/xdma/tsn.c

```diff
@@ -423,30 +416,22 @@ static bool get_timestamps(struct timestamps* timestamps, const struct tsn_confi
 	return true;
 }
 
-static bool append_buffer_track(struct buffer_tracker* buffer_tracker) {
-	if (buffer_tracker->pending_packets >= HW_QUEUE_SIZE) {
-		return false;
+static void tsn_buffer_tracker_update(struct xdma_dev* xdev) {
```

Comment:
```suggestion
static void update_tsn_buffer_tracker(struct xdma_dev* xdev) {
```

tsn_buffer_tracker를 함수명의 앞에 두어야하는 규칙이 정해져있는 것 일수도 있으니, 만약 그러하다면 이대로 사용하셔도 무방합니다.